functionality/remove-from-wishlist.js

```diff
@@ -0,0 +1,4 @@
+function showCodeForComp(id) {
```

Comment:
JS implemented does not seem to be functional
also might not be required for the current deadline

Reply:
It is working ... it is just that all cards are same so the cards seems to be deleting from last. Anyway JS not mandatory for now so keeping that for later.

Reply:
Noted, did not see the bottom  most row. Will take back that comment, works great!